guide/sequelize/currency.md

```diff
@@ -4,7 +4,7 @@ A common feature of Discord bots is a currency system. It's possible to do every
 
 ## File overview
 
-There will be multiple files: a DB init script, your models, and your bot script. In the previous tutorial, we placed all of these in the same file. Having everything in one file is usually not how it's done in the real world, so we'll correct that.
+There will be multiple files: a DB init script, your models, and your bot script. In the previous tutorial, we placed all of these in the same file. Having everything in one file is usually not how the real world does it, so we'll correct that.
```

Comment:
The original sentence reads better imo. But I'd like to stray away from phrases like "the real world". Suggestions welcome.

Reply:
Maybe something pertaining to practices? Ex.

> ... usually not a good practice ...

Reply:
```suggestion
There will be multiple files: a DB init script, your models, and your bot script. In the previous tutorial, we placed all of these in the same file. Having everything in one file isn't an ideal practice, so we'll correct that.
```